jme3-core/src/main/java/com/jme3/app/SimpleApplication.java

```diff
@@ -92,6 +93,8 @@ public void onAction(String name, boolean value, float tpf) {
                 if (stateManager.getState(StatsAppState.class) != null) {
                     stateManager.getState(StatsAppState.class).toggleStats();
                 }
+            }else if (name.equals(INPUT_RESTART_CONTEXT)){
```

Comment:
The new input binding is for any testing anyone wants to do. Just let me know when we are ready to merge, and I'll remove it and put it in TestContextRestart.